src/main/java/services/ImageTransferService.java

```diff
@@ -103,9 +103,12 @@
  *    transfer = transfer_service.get()
  * ----
  *
- * At that stage, if the transfer's phase is xref:types-image_transfer_phase[paused_system], then the session was
- * not successfully established. One possible reason for that is that the ovirt-imageio-daemon is not running
- * in the host that was selected for transfer.
+ * At that stage, if the phase of the transfer is <<types/image_transfer_phase, paused_system>>, the session was
```

Comment:
The format on internal links has changed recently, please keep: xref:types-image_transfer_phase[paused_system]